README.md

```diff
@@ -35,18 +35,30 @@ your Cloud SQL instance.
 The instance connection name for your Cloud SQL instance is always in the
 format "project:region:instance".
 
+### APIs and Services
+
+This package requires that the [Cloud SQL Admin API][admin-api] be enabled
+within your Google Cloud Project.
+
+[admin-api]: https://console.cloud.google.com/apis/api/sqladmin.googleapis.com
+
 ### Credentials
 
 This repo uses the [Application Default Credentials (ADC)][adc] strategy for
-typing providing credentials. Please see the
+resolving credentials. Please see the
 [golang.org/x/oauth2/google][google-auth] documentation for more information in
 how these credentials are sourced.
 
 To explicitly set a specific source for the Credentials to use, see [Using
 Option](#using-options) below.
 
+The source identity of the credentials (user, service account, etc.) must have
+the [Cloud SQL Client][client-role] IAM Role in order to successfully 
+make Cloud SQL connections.
+
```

Comment:
Can we figure out how to work this higher? Essentially we want to lead with "You must have a IAM member with this role" then say "here's how we figure out which IAM member we use".